charts/rancher-webhook/2.0.0+up0.3.0-rc3/Chart.yaml

```diff
@@ -8,11 +8,11 @@ annotations:
   catalog.cattle.io/rancher-version: '>= 2.7.0-0 < 2.8.0-0'
   catalog.cattle.io/release-name: rancher-webhook
 apiVersion: v2
-appVersion: 0.3.0-rc2
```

Comment:
This asset needs to get deleted: https://github.com/rancher/charts/tree/dev-v2.7/assets/rancher-webhook

Reply:
I believe it was deleted in the make charts commit.
git just registered it as a rename.

Reply:
Oh, missed that, usually I just look for two asset changes.